pipeline_dp/combiners.py

```diff
@@ -20,6 +20,7 @@
 from pipeline_dp import budget_accounting
 import numpy as np
 import collections
+from pipeline_dp.aggregate_params import NormKind
```

Comment:
According to Google styleguide, it's needed to import modules, not classes. Could you please
1.Add NormKind to [__init__.py](https://github.com/OpenMined/PipelineDP/blob/main/pipeline_dp/__init__.py) (there are public API objects).
2. Use `pipelinedp.NormKind` in the code